.deploy.sh

```diff
@@ -0,0 +1,15 @@
+#/bin/bash
+
+usage() {
+    echo -e "\tUSAGE: $0 USERNAME PASSWORD SERVER"
+}
+
+if [ $# -lt 3 ]; then
+    echo "ERROR: missing arguments"
+    usage
+    exit 2
+fi
+
+echo "Updating Files"
+
+curl --user "$1:$2" -T . "$3"
```

Comment:
[Upload to dev folder in WebDAV to test](https://faq.its.fsu.edu/web-services/webdav-departments)